content/en/real_user_monitoring/guide/monitor-hybrid-react-native-applications.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Monitor Hybrid React Native Applications
+kind: guide
+description: Guide for monitoring hybrid React Native applications.
+further_reading:
+- link: '/monitors/create/types/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn about RUM Monitors'
+---
+
+## Overview
+
+If you have a hybrid application built on React Native, you can use Datadog to monitor the same application both from the React Native and native sides.
```

Comment:
What is "React Native" vs just "native" here?

Reply:
added description